engine/runtime/src/test/scala/org/enso/interpreter/test/semantic/SystemProcessTest.scala

```diff
@@ -131,11 +124,10 @@ class SystemProcessTest extends InterpreterTest with OsSpec {
       val input = Random.nextBytes(Byte.MaxValue)
       val code =
         """import Standard.Base.System
-          |import Standard.Base.Data.Array.Array
           |from Standard.Base.Data.Boolean import all
           |
           |main =
-          |    result = System.create_process "bash" (Array.new_2 "-c" "wc -c") "" True True True
+          |    result = System.create_process "bash" ["-c", "wc -c"] "" True True True
```

Comment:
Now the syntax looks like a normal syntax in normal language - way better than `Array.new_2`!